src/gui/properties/peerlistwidget.cpp

```diff
@@ -433,7 +443,7 @@ void PeerListWidget::updatePeer(const BitTorrent::Torrent *torrent, const BitTor
     const Qt::Alignment intDataTextAlignment = Qt::AlignRight | Qt::AlignVCenter;
 
     const auto setModelData =
-        [this] (const int row, const int column, const QString &displayData
+        [this](const int row, const int column, const QString &displayData
                 , const QVariant &underlyingData, const Qt::Alignment textAlignmentData = {}
                 , const QString &toolTip = {})
```

Comment:
I think the 2 lines above need to have a space deleted too. (to align correctly)

Reply:
>to align correctly

What is correct alignment? Currently they are shifted by several count of indentations (so their overall indentation is power of 4 spaces).

Reply:
I think it is customary to begin one character after the opening parentheses in this case.

Reply:
> I think it is customary to begin one character after the opening parentheses in this case.

We don't have strict rules about this. Therefore, I would prefer not to dwell on this here, but to discuss it separately, if that's what you care about.
I'll just say right away that such "conditional" alignments make the code terrible in some (very frequent) cases, since it can leave too much unjustifiably unused space on the left, unlike the fixed-indent method.